aeppl/transforms.py

```diff
@@ -135,6 +158,63 @@ def default_transform(fgraph: FunctionGraph, node: Node) -> Optional[List[Node]]
     return trans_node.outputs
 
 
+class TransformValuesMapping(Feature):
+    r"""A `Feature` that maintains a map between value variables and their transforms."""
+
+    def __init__(self, values_to_transforms):
+        self.values_to_transforms = values_to_transforms
+
+    def on_attach(self, fgraph):
+        if hasattr(fgraph, "values_to_transforms"):
+            raise AlreadyThere()
+
+        fgraph.values_to_transforms = self.values_to_transforms
+
+
+class TransformValuesOpt(GlobalOptimizer):
+    r"""Transforms value variables according to a map and/or per-`RandomVariable` defaults."""
+
+    default_transform_opt = in2out(default_transform, ignore_newtrees=True)
+
+    def __init__(
+        self,
+        values_to_transforms: Dict[
+            TensorVariable, Union[RVTransform, DefaultTransformSentinel, None]
+        ],
+    ):
+        """
+        Parameters
+        ==========
+        values_to_transforms
+            Mapping between value variables and their transformations.
+            Each value variable can be assigned one of `RVTransform`,
+            ``DEFAULT_TRANSFORM``, or ``None``. If a transform is not specified for
+            a specific value variable it will not be transformed.
+
+        Raises
+        ======
+        RuntimeError
```

Comment:
It's just a warning now

Reply:
Yes, by design; it's not clear that an error is more suitable.

Reply:
I meant that the docstrings are incorrect now